[opentitantool] Conf files relative to current directory

jesultra opened the pull request:
Daemons should generally not keep a reference to the current directory as of when they were
started, doing so runs the risk of random references preventing the unmounting or other management
of filesystem volumes, due to some accident in the way a daemon was started.  For that reason, the
opentitansession daemon it started with the root directory ( / ) as its current directory.

However, the first business of the daemon process is to bind to a socket, and read configuration
files according to command line arguments.  Those arguments could include relative paths to some
files.  In order to support that, this CL delays switching to the root directory until the daemon
process has finished initializing, and at the same time when it drops its connection to the
terminal that was used to start it.

Signed-off-by: Jes B. Klinke <jbk@chromium.org>
Change-Id: Ia3216568b382ca04c12c7613822343052daefbe1

mcy commented on sw/host/opentitansession/src/main.rs:
```diff
@@ -163,6 +162,11 @@ fn session_child(listen_port: Option<u16>, backend_opts: &backend::BackendOpts)
+    // All configuration files have been processed (relative to current direction), we can now
+    // drop the reference to the file system, (in case the admin wants to unmount while this
+    // daemon is still running.)
+    env::set_current_dir(&Path::new("/"))?;
```
Nit: Since this actually takes `impl AsRef<Path>`, you could also pass in just `"/"`. Most things that take a Path are like this; similarly, the & is unnecessary; `Path:;new()` actually looks like
```rust
pub fn new<S: AsRef<OsStr> + ?Sized>(s: &S) -> &Path;
```

jesultra replied:
Thanks, done.

jesultra commented:
As I am back in office, I moved my OpenTitan git checkout from my laptop to my workstation (by means of cpio).  Since doing so, all my PRs get rejected with this message:
```
Commit has one or more Signed-off-by lines, but not the one we expect. We expected to find "Signed-off-by: {Jes B. Klinke} <jbk@google.com>".
```
I have so far used my jbk@chromium.org account in the Signed-off-by lines, and would like to be able to continue to do so.
My question is, where does the jbk@google.com (and strange curly brackets around my name) come from?  I have added a new SSH key from my workstation to my Github account, maybe that contains the email that commits are matched against.

timothytrippel commented:
Looks like your latest push has resolved the issue. May I merge this?
